web-frontend/src/main/v3/packages/ui/src/components/Error/ErrorDetailDialog.tsx

```diff
@@ -48,15 +48,23 @@ export const ErrorDetailDialog = ({
               <div className="w-1 h-4 rounded-sm bg-status-fail" />
               Error Details
             </h4>
-            <p className="text-sm font-semibold text-muted-foreground">{error?.instance}</p>
+            <div className="flex items-center gap-1">
+              <a
+                className="text-sm font-semibold text-primary hover:underline"
+                href={error?.url}
```

Comment:
if error type is `error: ErrorResponse & { url: string };`,  optional type is not necessary

or correct error type to `error: ErrorResponse & { url?: string };`

Reply:
Added new type `ErrorDetailResponse` and corrected error type.